src/java.base/share/conf/security/java.security

```diff
@@ -94,6 +94,7 @@ security.provider.tbd=SunPKCS11
 RestrictedSecurity.NSS.140-2.desc.name = Red Hat Enterprise Linux 8 NSS Cryptographic Module FIPS 140-2
 RestrictedSecurity.NSS.140-2.desc.default = true
 RestrictedSecurity.NSS.140-2.desc.fips = true
+RestrictedSecurity.NSS.140-2.fips.mode = 140-2
```

Comment:
I think this should be added after the set of descriptive properties; likewise for FIPS140-3.

Reply:
Makes sense to me too. Moved this property for both the NSS and OpenJCEPlus profiles to the end of the description section. I also added a new line right after the description section in the OpenJCEPlus profile to segment that profile similar to NSS.